src/lang/english.txt

```diff
@@ -1258,6 +1259,9 @@ STR_CONFIG_SETTING_DYNAMIC_ENGINES_EXISTING_VEHICLES            :{WHITE}Changing
 STR_CONFIG_SETTING_INFRASTRUCTURE_MAINTENANCE                   :Infrastructure maintenance: {STRING2}
 STR_CONFIG_SETTING_INFRASTRUCTURE_MAINTENANCE_HELPTEXT          :When enabled, infrastructure causes maintenance costs. The cost grows over-proportional with the network size, thus affecting bigger companies more than smaller ones
 
+STR_CONFIG_SETTING_COMPANY_STARTING_COLOUR                      :Starting company colour: {STRING2}
```

Comment:
My comment feels a bit like bike shedding, yet: AI also start with a company colour. Maybe "Default company colour" with a help text of "Choose a company colour the player starts with by default" would make it ever so slightly clearer?

Reply:
Yes, it sound better, I will change it. It is always hard to make a short and clear description, so thanks for the help.